app/src/main/java/me/ccrama/redditslide/Views/ToolbarColorizeHelper.java

```diff
@@ -122,7 +130,8 @@ public void onGlobalLayout() {
         });
     }
 
-    private static void removeOnGlobalLayoutListener(View v, ViewTreeObserver.OnGlobalLayoutListener listener) {
+    private static void removeOnGlobalLayoutListener(View v,
+            ViewTreeObserver.OnGlobalLayoutListener listener) {
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
The diff apparently does not care that this is part of the diff, but in these easy cases, I just call the more appropriate method for pre-Jelly-Bean since these were just syntax changes and not actual API changes.